src/utils.ts

```diff
@@ -185,6 +207,18 @@ export namespace Utils {
       );
     }
 
+    // verify homeDomain
+    if (homeDomain) {
+      const homeDomainKeyValue = homeDomain.startsWith("https://")
```

Comment:
there seems to be some inconsistency between the code which extracts the home domain value here and the code in `buildChallengeTx()`